### ISSUE TITLE
In editor make description of editor fields accessible

### ISSUE THREAD

**Issue by odzhychko** [nextcloud/calendar#8887]
### Is your feature request related to a problem? Please describe.

<img width="2269" height="503" alt="Image" src="https://github.com/user-attachments/assets/e4d602a1-1ad2-4db7-9478-a86905dc5318" />

We have descriptions of editor fields, but they are only used in aria-lable and are not accessible through tooltip or the like when using the mouse.

### Describe the solution you'd like

1. When hovering on the icon of an editor field show a tooltip with the description.

### Describe alternatives you've considered

2. Info icon on the right of the input element
    - Discarded because it would litter UI with many info icons
3. Info icon inside the input element
    - Most input elements are dropdowns and already have the arrow down icon inside the input
4. Clicking on the icon opens a modal with the description
    - Our description are usually not so long to need a dedicated modal.

**Comment by odzhychko** [comment]
@kra-mo Maybe you have some input what a good UI/UX would be for the editor field descriptions.

**Comment by kra-mo** [comment]
If this is really a problem, I feel like this should be changed upstream instead.

But also, for the specific example, I would split it up like so:

<img width="356" height="58" alt="Image" src="https://github.com/user-attachments/assets/d36fe922-d1ee-420d-b546-c0aa349dd2d5" />

**Comment by kra-mo** [comment]
In general, using icons like this is not great. We should probably use titles instead, or at least in addition.
